src/main/java/com/Jacksonnn/DCEvents/Configuration/ConfigManager.java

```diff
@@ -64,6 +64,7 @@ public static void configCheck(ConfigType type) {
             config.addDefault("Events.CommandDescriptions.Teleport", "Teleports all active players in an event to the executor.");
             config.addDefault("Events.CommandDescriptions.ReloadCommand", "Reloads configuration files into the active instance of the plugin.");
             config.addDefault("Events.CommandDescriptions.JoinCommand", "Allows players to join events.");
+            config.addDefault("Events.CommandDescriptions.JoinCommand", "Allows players to leave an event they\'re apart of.");
```

Comment:
```suggestion
            config.addDefault("Events.CommandDescriptions.LeaveCommand", "Allows players to leave an event they're apart of.");
```
Typo, and no need to include a backslash unless the special character is a line break of double quotation mark.